Dockerfile

```diff
@@ -1,156 +1,86 @@
-
-# install more dependencies
-
-RUN apt install -y zlib1g-dev
-
-
-# build openssl
-
-WORKDIR /root/openssl
-# uname -m
-# well we could also simply use the prebuilt static libssl from libssl-dev on ubuntu ...
-RUN ./Configure no-shared --prefix=/static-prefix  linux-x86_64
-RUN make -j$(nproc)
-RUN make install
-
-
-# now build "the big one" (aka "qt5")
-
-WORKDIR /root/qt5
-
-# only checkout required submodules
-RUN ./init-repository --module-subset=qtbase
-
-ENV MAKEFLAGS="-j 4"
-RUN ./configure -confirm-license -opensource -v \
-	-prefix /static-prefix \ 
-	-plugin-sql-sqlite \
-	-nomake examples \
-	-nomake tests \
-	-skip qt3d \
-	-skip qtactiveqt \
-	-skip qtandroidextras \
-	-skip qtcanvas3d \
-	-skip qtcharts \
-	-skip qtconnectivity \
-	-skip qtdatavis3d \
-	-skip qtdeclarative \
-	-skip qtdoc \
-	-skip qtdocgallery \
-	-skip qtfeedback \
-	-skip qtgamepad \
-	-skip qtgraphicaleffects \
-	-skip qtimageformats \
-	-skip qtlocation \
-	-skip qtlottie \
-	-skip qtmacextras \
-	-skip qtmultimedia \ 
-	-skip qtnetworkauth \
-	-skip qtpim \
-	-skip qtpurchasing \
-	-skip qtqa \
-	-skip qtquick3d \
-	-skip qtquickcontrols \
-	-skip qtquickcontrols2 \
-	-skip qtquicktimeline \
-	-skip qtremoteobjects \
-	-skip qtrepotools \ 
-	-skip qtscript \
-	-skip qtscxml \
-	-skip qtsensors \
-	-skip qtserialbus \
-	-skip qtserialport \
-	-skip qtspeech \
-	-skip qtsvg \
-	-skip qtsystems \
-	-skip qttranslations \
-	-skip qtvirtualkeyboard \
-	-skip qtwayland \
-	-skip qtwebchannel \
-	-skip qtwebengine \
-	-skip qtwebglplugin \
-	-skip qtwebsockets \
-	-skip qtwebview \
-	-skip qtwinextras \
-	-skip qtx11extras \
-	-skip qtxmlpatterns \
-	-no-gui \
-	-no-widgets \
-	-no-dbus \
-	-no-strip \
-	-openssl-linked \
-	-I /static-prefix/include \
-	-L /static-prefix/lib \
-	-static \
-	-no-shared
-
-RUN make -j$(nproc)
-RUN make install
-
-
-# install further dependencies required to compile mumble
-
-RUN apt install -y libprotobuf-dev libcap-dev protobuf-compiler protobuf-c-compiler libboost1.74-dev
-
-
-# build mumble/murmur
+# =================================
+# Build Enviroment
+# =================================
 
+FROM ubuntu:24.04 AS build_environment
+RUN apt update && apt install -y wget xz-utils
+
+ENV MUMBLE_ENVIRONMENT_VERSION=mumble_env.x64-linux.2025-02-27.d7001f6639
+
+# download pre-built environment
+RUN wget -q -O env.tar.xz "https://github.com/mumble-voip/vcpkg/releases/download/2025-02/$MUMBLE_ENVIRONMENT_VERSION.tar.xz"
+
+# extract environment
+RUN tar -xf env.tar.xz
+RUN mv mumble_env.* /mumble-build-environment
+
+
+# =================================
+# Mumble
+# =================================
+FROM ubuntu:24.04 AS mumble_builder
+
+RUN apt update && apt install -y git cmake vim file ninja-build g++-multilib python3
+
+# additional dependencies
+RUN apt-get install -y libdbus-1-dev libsystemd-dev
+
+# download mumble
+ENV MUMBLE_VERSION=05b4c95b 
```

Comment:
Is this a stable version? I could not find any info about mumble 1.6 by quickly checking the project.
How close is this commit to a mumble-1.6 release of the server part (murmur)?

Reply:
this is just a random commit being the latest in that moment I was working on it. Stable 1.6 releases might be coming this year (see post below)